Tasks/Maven/maventask.ts

```diff
@@ -170,11 +186,121 @@ function publishJUnitTestResults(testResultsFiles: string) {
         tl.warning('No test result files matching ' + testResultsFiles + ' were found, so publishing JUnit test results is being skipped.');
         return 0;
     }
-    
+
     var tp = new tl.TestPublisher("JUnit");
     tp.publish(matchingJUnitResultFiles, true, "", "", "", true);
 }
 
+function enableCodeCoverage() {
+    var classFilter = tl.getInput('classFilter');
+    var classFilesDirectories = tl.getInput('classFilesDirectories');
+    var sourceDirectories = tl.getInput('srcDirectories');
+    var buildRootPath = path.dirname(mavenPOMFile);
+    // appending with small guid to keep it unique. Avoiding full guid to ensure no long path issues.
+    var reportPOMFileName = "CCReportPomA4D283EG.xml";
+    reportPOMFile = path.join(buildRootPath, reportPOMFileName);
+    var targetDirectory = path.join(buildRootPath, "target");
+    ccReportTask = "jacoco:report";
```

Comment:
Can you add type information to the above vars?
`var classFilter: string = tl.getInput('classFilter');`